projects/core/src/checkout/facade/checkout-delivery.service.ts

```diff
@@ -154,11 +228,41 @@ export class CheckoutDeliveryService {
    * Clear address verification results
    */
   clearAddressVerificationResults(): void {
-    this.checkoutStore.dispatch(
-      new CheckoutActions.ClearAddressVerificationResults()
+    this.store.dispatch(new CheckoutActions.ClearAddressVerificationResults());
+  }
+
+  /**
+   * Clear address already setup in last checkout process
+   */
+  clearCheckoutDeliveryAddress(): void {
```

Comment:
Should we combine this into one method `clearCheckoutDeliveryDetails`?

Reply:
that is fascade layer, I don't think so its a good idea